sdk/storage/azure-storage-blobs/src/blob_service_client.cpp

```diff
@@ -52,8 +52,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     }
     policies.emplace_back(std::make_unique<StoragePerRetryPolicy>());
     policies.emplace_back(std::make_unique<SharedKeyPolicy>(credential));
-    policies.emplace_back(std::make_unique<Azure::Core::Http::TransportPolicy>(
-        std::make_shared<Azure::Core::Http::CurlTransport>()));
+    policies.emplace_back(std::make_unique<Azure::Core::Http::TransportPolicy>());
```

Comment:
Doesn't this need the transport options passed to it?

Maybe we should suggest to the Storage team that they have some internal function that creates the pipeline and then they just call it from the various ctors. 

Now, a change to pipeline requires many changes throughout the code base. This is brittle